proposals-CG-WG.md

```diff
@@ -1,82 +1,78 @@
+  * What's asked of the **Working Group**?
+    * Working Group consensus to adopt the proposal as the basis for their work in the [Editor’s Draft](https://w3c-fedid.github.io/FedCM/).
+    * The Working Group identifies a list of (seemingly resolvable) issues that have to be addressed before [Stage 3](#stage-3).
+  
+# Stage 3: Independent Implementations
 
-# Stage 2
-
-The purpose of Stage 2 Proposals is to formally specify the best (and seemingly workable) alternative that was identified in the prior step: handle corner cases, integrate with other parts, reconcile with other proposals and resolve the concerns identified at the entrance. The Proposal enters Stage 2 with a list of blocking issues to advance to the next stage and exits with all of the issues resolved.
-
-  * Entrance criteria
-    * Products of the prior stage and
-    * WG consensus to adopt the solution as the basis for their work.
-    * The WG identifies a list of (seemingly resolvable) issues that have to be addressed before Stage 3.
-  * Entrance artifact
-    * The spec PR gets merged into the [Editor’s Draft](https://w3c-fedid.github.io/FedCM/)
-    * The feature repo gets transferred to the Working Group 
-  * Exit criteria
+ The purpose of Stage 3 Proposals is to increase implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
+ 
+  * What's needed from champions?
     * Proposal fully merged into the [Editor’s Draft](https://w3c-fedid.github.io/FedCM/)
-    * The [Editor’s draft](https://www.w3.org/policies/process/#editors-draft) addresses all issues that were identified at the entrance
     * Web Platform Tests
     * Further implementation experience
-    
-# Stage 3
-
-The purpose of Stage 3 Proposals is to increase the implementation and deployment confidence in order to produce a [Candidate Recommendation](https://www.w3.org/policies/process/#RecsCR).
+  * What's asked of the **Working Group**? 
+    * Working Group consensus that the [Editor’s Draft](https://www.w3.org/policies/process/#editors-draft) sufficiently resolves all of the issues raised at [Stage 2](#stage-2)
+  * What happens if accepted?
+    * The [Editor’s Draft](https://w3c-fedid.github.io/FedCM/) gets published as the Working Group's [Working Draft](https://www.w3.org/TR/fedcm/)
+  
+# Stage 4: Publication
 
-  * Entrance criteria
-    * Products of the prior stage and
-    * WG consensus that the [Editor’s Draft](https://www.w3.org/policies/process/#editors-draft) sufficiently resolves all of the issues raised
-  * Entrance artifact
-    * The [Editor’s Draft](https://w3c-fedid.github.io/FedCM/) gets published as the WG [Working Draft](https://www.w3.org/TR/fedcm/)
-  * Exit criteria
+ The purpose of Stage 4 Proposals is to produce a [W3C Recommendation](https://www.w3.org/policies/process/#RecsW3C).
+ 
+  * What's needed from champions?
     * Two independent and interoperable implementations
```

Comment:
This will be a critical requirement so it is important to be precise. Do two Chromium browsers count? Or does it need to be implemented by two independent rendering engines?

Reply:
Good question, but will leave that clarification to a separate PR with other reviewers too. It is ultimately the bar that the W3C sets, I think.

Reply:
Ok, resolving this in this PR:

https://github.com/w3c-fedid/Administration/pull/9

Resolving this here.